src/components/InputModal.tsx

```diff
@@ -236,6 +234,7 @@ export default class InputModal extends React.Component<
     if (code === 13) {
       if (this.state.isValid) {
         this.props.onModalClose(this.state.input, this.state.savedInput);
+        await this.setState({ modalOpen: false, input: "" });
```

Comment:
why block here?

Reply:
Good point, the await there is probably not needed.